subprojects/launcher/src/main/java/org/gradle/launcher/daemon/configuration/DaemonBuildOptions.java

```diff
@@ -30,35 +31,28 @@
 import org.gradle.process.internal.JvmOptions;
 
 import java.io.File;
-import java.util.ArrayList;
-import java.util.Collections;
 import java.util.List;
 import java.util.Locale;
 
 public class DaemonBuildOptions extends BuildOptionSet<DaemonParameters> {
 
-    private static List<BuildOption<DaemonParameters>> options;
-
-    static {
-        List<BuildOption<DaemonParameters>> options = new ArrayList<BuildOption<DaemonParameters>>();
-        options.add(new IdleTimeoutOption());
-        options.add(new HealthCheckOption());
-        options.add(new BaseDirOption());
-        options.add(new JvmArgsOption());
-        options.add(new JavaHomeOption());
-        options.add(new DebugOption());
-        options.add(new DebugHostOption());
-        options.add(new DebugPortOption());
-        options.add(new DebugServerOption());
-        options.add(new DebugSuspendOption());
-        options.add(new ApplyInstrumentationAgentOption());
-        options.add(new DaemonOption());
-        options.add(new ForegroundOption());
-        options.add(new StopOption());
-        options.add(new StatusOption());
-        options.add(new PriorityOption());
-        DaemonBuildOptions.options = Collections.unmodifiableList(options);
-    }
+    private static List<BuildOption<DaemonParameters>> options = ImmutableList.of(
```

Comment:
❓   Is is a problem to use `ImmutableList` in launcher? It may be if our Guava version requires Java 8.

Reply:
as discussed:
SupportedBuildJvmIntegrationTest runs green, so we are fine here.